### PR TITLE
sqlite3: add system libs to FreeBSD too

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **sqlite3/***

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [X] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/sqlite3/all/conanfile.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def package_info(self):
         self.cpp_info.components["sqlite"].names["cmake_find_package"] = "SQLite3"
         self.cpp_info.components["sqlite"].names["cmake_find_package_multi"] = "SQLite3"
         self.cpp_info.components["sqlite"].libs = tools.collect_libs(self)
-        if self.settings.os == "Linux":
+        if self.settings.os == "Linux" or self.settings.os == "FreeBSD":
```
Comment (SpaceIm) [inline recipes/sqlite3/all/conanfile.py:135]:
```suggestion
        if self.settings.os in ["Linux", "FreeBSD"]:
```

**Comment by madebr** [comment]
@ericLemanissier 
How did you catch this error?
If you're testing FreeBSD, can you take a look at #3815?

**Comment by ericLemanissier** [comment]
I did not, This is @patricia-gallardo's work too !

**Comment by patricia-gallardo** [comment]
> I did not, This is @patricia-gallardo's work too !

Thankfully @ericLemanissier takes my bug reports on twitter DM - I'm too shy for this 🤣

**Review by uilianries — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**